weave

```diff
@@ -847,7 +849,7 @@ case "$COMMAND" in
             -p $PORT:$CONTAINER_PORT/tcp -p $PORT:$CONTAINER_PORT/udp \
             -v /var/run/docker.sock:/var/run/docker.sock \
             -e WEAVE_PASSWORD \
-            $WEAVE_DOCKER_ARGS $IMAGE -iface $CONTAINER_IFNAME -port $CONTAINER_PORT -name "$PEERNAME" -nickname "$(hostname)" $IPRANGE "$@")
+            $WEAVE_DOCKER_ARGS $IMAGE -iface $CONTAINER_IFNAME -port $CONTAINER_PORT -name "$PEERNAME" -nickname "$(hostname)" $IPRANGE $ARGS)
```

Comment:
Thanks! Based on the [docs](http://pubs.opengroup.org/onlinepubs/009695399/utilities/set.html) and experiments, the 0-arg guard is unnecessary:

> The command set -- without argument shall unset all positional parameters and set the special parameter '#' to zero.